counterparty-core/counterpartycore/protocol_changes.json

```diff
@@ -637,5 +637,12 @@
         }
       }
     }
+  },
+  "accept_only_one_valid_dispenser": {
+    "minimum_version_major": 11,
+    "minimum_version_minor": 1,
+    "minimum_version_revision": 0,
+    "block_index": 966000,
```

Comment:
Let's make this for one week from today.

Reply:
"block_index": 868200